tap_adroll/client.py

```diff
@@ -41,6 +53,17 @@ def __init__(self, config_path, config):
             LOGGER.info("Error initializing AdrollClient during token refresh, please reauthenticate.")
             raise AdrollAuthenticationError(e)
 
+    def _read_config(self):
+        """
+        Performs read on the provided filepath,
+        returns empty dict if invalid path provided
+        """
+        try:
+            with open(self.config_path,'r') as file:
+                return json.load(file)
+        except FileNotFoundError as _:
+            LOGGER.fatal("Failed to load config in dev mode")
+            return {}
```

Comment:
This implementation to return an empty dictionary and then let the key error above cause the tap to exit could be better.

I think a tap should not run without a valid config. And so if this function revealed the fact that we have a bad token, I think it should cause the tap to fail and not wait for some other function to do it

Reply:
Removed this function as it is not needed.